jbpm-system-test/jbpm-clustered-ejb-timers-test/pom.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-system-test</artifactId>
+    <version>7.56.0.Final</version>
+  </parent>
+
+  <artifactId>jbpm-clustered-ejb-timers-test</artifactId>
+  
+  <name>Clustered EJB Timers</name>
+  <description>System tests using clustered EJB timers</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <junit.version>5.7.0</junit.version>
+
+    <!-- maven plugin versions -->
+    <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+    <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+
+    <!--this is used by logback-test.xml -->
+    <testLogRootLevel>info</testLogRootLevel>
+    <testcontainers.version>1.15.2</testcontainers.version>
+    <assertj.version>3.15.0</assertj.version>
+    <hamcrest.version>2.2</hamcrest.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <logback.version>1.2.3</logback.version>
+    <postgresql.driver.version>42.2.18</postgresql.driver.version>
+    <hikaricp.version>3.4.5</hikaricp.version>
```

Comment:
Don't we want to have this hooked up to the `kie-parent`? It could simplify the version management, for example.

Reply:
yes, it could help, added